README.md

```diff
@@ -16,9 +16,11 @@ The St. Olaf community, now in pocket size… rewritten in React Native.
 
 - [Install React Native](http://facebook.github.io/react-native/docs/getting-started.html#content)
 - [Install Yarn](https://yarnpkg.com/en/docs/install)
+- [Install Git Large File Storage](https://help.github.com/articles/installing-git-large-file-storage/)
 - Clone the repository
 - `cd` into your clone
 - Run `yarn`
+- Run `git lfs pull`
```

Comment:
Add `git lfs install` first. I managed to not install it system-wide, so let's be on the safe side.

Reply:
I think it only needs to be run once, but if this is peoples' first time using it, they should run that.  I think that since it adds LFS as a hook, cloning a LFS repository will probably also automatically pull them, but since these two operations are non-destructive, we don't need to worry about them.

Reply:
That's part of the installation instructions in the link.

Reply:
@rye are you saying we should not include instructions for this at all?

Reply:
I'm thinking that we could include the os x install instructions inline, since they're `brew install git-lfs` and `git lfs install`

Reply:
and then link to the installation page for other options

Reply:
@drewvolz No, I would say we should include this step to make sure that people have that.